content/projects/2022-brain-nets/index.md

```diff
@@ -4,6 +4,133 @@ date: 2022-06-27T11:36:49-04:00
 lastmod: 2022-06-27T11:36:49-04:00
 featured: true
 weight : 100
-summary: 'Orchestrating behavior and cognition, human brains lie at the core of complex neurobiological systems. Understanding of the structures, functions and mechanisms inside the brains has been an intriguing pursuit throughout human history. Recent studies in neuroscience and brain imaging have reached the consensus that the interactions among brain regions are driving factors for neural development and disorders, but there has not been an understanding in what mathematical models should be used to analyze such interactions, a.k.a the brain networks. In this project, we will explore and analyze different approaches for the modeling of brain networks, especially ranging from traditional shallow graph models to modern deep graph neural networks, towards the analysis of mental disorders, such as PTSD, depression and substance misuse. We will adapt different graph mining techniques for brain networks, statistically and visually analyze the results, and quantitatively evaluate them in the standard graph classification setting.'
+summary: 'Orchestrating behavior and cognition, human brains lie at the core of complex neurobiological systems. Understanding the structures, functions, and mechanisms inside the brain is the main goal of modern neuroscience research. Recent studies in neuroscience and brain imaging have reached the consensus that the interactions among brain regions are driving factors for neural development and disorders, but determining what types of mathematical models should be used to analyze such interactions is a current area of research. In this project, we explore and analyze different approaches for modeling brain networks, ranging from traditional shallow graph models to modern deep graph neural networks. The goal of these models is to aid in the analysis of mental disorders such as PTSD, depression, and substance misuse and to harness modern computational methods to improve classification accuracy of pre-existing models that aim to predict whether a brain is diseased or healthy. We will adapt different graph mining techniques for brain networks, statistically and visually analyze the results, and quantitatively evaluate them in the standard graph classification setting.'
 tags: ["Summer 2022"]
 ---
+# Overview: Comparing Shallow vs. Deep Brain Network Models
+
+We are comparing and benchmarking the performance of graph kernels and graph neural networks applied to disease classification based on neuroimaging data.
```

Comment:
Do you want to specify who is 'we'? I'll use the first paragraph later to link your paper and other materials you've created.